stdlib/public/stubs/FoundationHelpers.mm

```diff
@@ -110,15 +112,15 @@ static CFRange cast(_swift_shims_CFRange value) {
 _swift_shims_CFStringRef
 swift::_swift_stdlib_CFStringCreateCopy(_swift_shims_CFAllocatorRef alloc,
                                         _swift_shims_CFStringRef theString) {
-  return cast(CFStringCreateCopy(cast(alloc), cast(theString)));
+  return cast(CFStringCreateCopy(kCFAllocatorSystemDefault, cast(theString)));
 }
 
 _swift_shims_CFStringRef
 swift::_swift_stdlib_CFStringCreateWithBytes(
     _swift_shims_CFAllocatorRef _Nullable alloc, const uint8_t *bytes,
     _swift_shims_CFIndex numBytes, _swift_shims_CFStringEncoding encoding,
     _swift_shims_Boolean isExternalRepresentation) {
-  return cast(CFStringCreateWithBytes(cast(alloc), bytes, numBytes,
+  return cast(CFStringCreateWithBytes(kCFAllocatorSystemDefault, bytes, numBytes,
```

Comment:
Same

Reply:
Can we remove the CFAllocatorRef argument entirely from the shim?

Reply:
(I was assuming shims were ABI, but I guess not?)

Reply:
I'm pretty sure they're ABI.

Reply:
That'd be extremely unfortunate if they are. But if so, can we still replace the allocator argument with a `const void *unused` if we know we never use it?

Reply:
Sure